### ISSUE TITLE
[FEAT] Support for Windows GPU version of Deepstack

### ISSUE THREAD

**Issue by guarddog13** [jakowenko/double-take#224]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
As the title says support for GPU of Deepstack on Windows.

**Comment by jakowenko** [comment]
Is this related to the [DeepStack addon](https://github.com/jakowenko/double-take-hassio-addons)? I can look into publishing the GPU version as well.

**Comment by guarddog13** [comment]
Yes sorry that's what I meant. I can get the docker GPU version running and double take sees that it is attached but it returns nothing when I try to click on matches in Double Take. Thank you so much!!

**Comment by guarddog13** [comment]
@jakowenko I now use deepstack on a separate Ubuntu machine that has an Nvidia GPU. I set deepstack gpu up on this machine with docker. Does doubletake not support the GPU version of deepstack? When I attempt to train with double take i get a 500 error...training with the same picture on deepstack non-gpu it gives a positive result. Double take and frigate using the gpu works fine.

**Comment by jakowenko** [comment]
> @jakowenko I now use deepstack on a separate Ubuntu machine that has an Nvidia GPU. I set up deepstack gpu up on this machine with docker. Does doubletake not support the GPU version of deepstack? When I attempt to train with double take i get a 500 error...training with the same picture using deepstack non-gpu gives a positive result. 
> 
> 
> 
> FYI Double take and Frigate GPU works fine. 

Hm...do you get back anything else besides the 500? It's most likely something with DeepStack. The API should work the same across their different versions. Can you replicate the 500 with their examples on their docs? May need to open an issue with them and see if they know what's going on.
